digital_image_processing/change_brightness.py

```diff
@@ -0,0 +1,24 @@
+from PIL import Image
+
+
+def change_brightness(img: Image, level: float) -> Image:
```

Comment:
OK...  This looks ready to go.  Thanks for doing this.  Just a few more questions...
1. Should level be a float or an int?
2. What should be the minimum and maximum allowed values for level?
    - [ ] 0.0 to 1.0
    - [ ] -1.0 to 1.0
    - [ ] 0 to 100
    - [ ] 0 to 255
3. If level is 0 then is the resulting image black?
4. If the level is maxed then is the resulting image white or normal?

Reply:
1. float
2. -255.0 to 255.0
3. No, if level 0 nothing changes.
4. If the level is maximized, the resulting image is white